### PR TITLE
added changes to stripAnswer function

### PR DESCRIPTION
**Pull request by niccireinking**

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -41,12 +41,14 @@ public Datastore() {
 
     public void savePuzzles() {
         Puzzle puzzle1 = new Puzzle(1, Puzzle.Puzzle_Type.TEXT, "At midday, I am the same size and shape of an elephant, but I do not weight the same as an elephant. At night, I am not seen at all. What am I?",
-            "ELEPHANTSSHADOW");
+            "(ELEPHANT'?S?)?\\s?SHADOW");
```
Comment (miaboloix) [inline src/main/java/com/google/codeu/data/Datastore.java]: is this the fixed reg-ex? does it accept 'the', 'an', etc?

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -41,12 +41,14 @@ public Datastore() {
...
         storePuzzle(puzzle1);
         Puzzle puzzle2 = new Puzzle(2, Puzzle.Puzzle_Type.TEXT, "What animal do you not want to play cards with? (TIP: Upload a picture of the animal for your answer. Submit the file and then submit in the answer box.)", "CHEETAH");
         storePuzzle(puzzle2);
         Puzzle puzzle3 = new Puzzle(3, Puzzle.Puzzle_Type.TEXT, "Rearrange the words on the map to get the secret message.", "THE QUICK BROWN FOX JUMPS OVER THE LAZY DOG");
         storePuzzle(puzzle3);
+        Puzzle puzzle4 = new Puzzle(4, Puzzle.Puzzle_Type.TEXT, "Find the needle in the haystack.", "(47.6038321,-122.3300623)");
```
Comment (miaboloix) [inline src/main/java/com/google/codeu/data/Datastore.java:50]: change to level 5!

File: src/main/java/com/google/codeu/servlets/UserLevelServlet.java [hunk 3]
```diff
@@ -97,7 +99,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
+        boolean isCorrect = false;
+        if(userLevel == 1){
+            if(Pattern.matches(correctAnswer, userAnswer)){
+                isCorrect = true;
+            }
+        }
+        else{
+            if(correctAnswer.equals(userAnswer)){
+                isCorrect = true;
+            }
```
Comment (miaboloix) [inline src/main/java/com/google/codeu/servlets/UserLevelServlet.java]: looks good!

File: src/main/java/com/google/codeu/servlets/UserLevelServlet.java [hunk 4]
```diff
@@ -106,15 +123,7 @@ public String stripAnswer(String userAnswer){
...
-            userAnswer = userAnswer.substring(userAnswer.indexOf(' ') + 1);
-        }
-        //go through user answer to sript any punctation 
-        for(int i = 0; i < userAnswer.length(); i++){
-            char letter = userAnswer.charAt(i);
-            //check if character is letter or digit 
-            if(Character.isLetterOrDigit(letter)){
-                result += letter;
-            }
+            result = userAnswer.substring(userAnswer.indexOf(' ') + 1);
```
Comment (miaboloix) [inline src/main/java/com/google/codeu/servlets/UserLevelServlet.java:126]: i think you can do this w a reg ex!

Comment (miaboloix) [reply src/main/java/com/google/codeu/servlets/UserLevelServlet.java:126]: but if not this also works

File: src/main/webapp/js/level4-loader.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        map: map,
+        title: title
+    });
+    var infoWindow = new google.maps.InfoWindow({
+        content: description
+    });
+    marker.addListener('click', function() { //changed name from addEventListener to show description
+        infoWindow.open(marker.get('map'), marker);
+    });
+}
```
Comment (miaboloix) [inline src/main/webapp/js/level4-loader.js:38]: looks good!

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (niccireinking) [reply src/main/java/com/google/codeu/data/Datastore.java]: no this only accepts elephant('s) shadow

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 2 above)
Comment (niccireinking) [reply src/main/java/com/google/codeu/data/Datastore.java:50]: Done!

**Pull request closed without merging**